src/Debugging-Settings/DebugSystemSettings.class.st

```diff
@@ -4,7 +4,7 @@ Settings for debugging
 Class {
 	#name : #DebugSystemSettings,
 	#superclass : #Object,
-	#category : #'Debugger-Model-Settings'
+	#category : #'Debugging-Settings'
```

Comment:
Package renaming should include removal of the previous package and update of the baseline.

Reply:
This is not renaming, this is a new package.
Baseline updated.